nix/cardano/constants.nix

```diff
@@ -0,0 +1,29 @@
+{
+  inputs,
+  cell,
+}: {
+  node-snapshots = {
+    # https://updates-cardano-testnet.s3.ap-southeast-1.amazonaws.com/cardano-node-state/index.html
+    testnet = {
+      base_url = "https://updates-cardano-testnet.s3.ap-southeast-1.amazonaws.com/cardano-node-state/index.html";
+      file_name = "db-testnet.tar.gz";
+    };
```

Comment:
@jbgi Do you think we could make these constants emanate somehow out of `cardano-world` itself?

Reply:
you mean how the upload script could use constant from cardano-world for path and bucket?

Reply:
Even further, how they could be loaded from `cache.iog.io` &mdash; being just normal build artifacts?

If necessary, we create a dependency on: https://github.com/numtide/nar-serve